extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelInitializedReactiveExecutorBuildItem.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.core.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.runtime.RuntimeValue;
+import org.apache.camel.spi.ReactiveExecutor;
+
+/**
+ * Holds the initialized {@link ReactiveExecutor} {@link RuntimeValue}.
```

Comment:
I'd appreciate a bit more explanations here: What does "initialized" mean here? Esp. what does that mean for the potential consumers of this BuildItem? When should they consume this one rather then the plain CamelReactiveExecutorBuildItem?

Similarly the CamelReactiveExecutorBuildItem should give hints when to use that one vs. this one.